.env.example

```
@@ -54,6 +54,13 @@ VERIFIER_REDIS_TIMEOUT_MS=2000
 
 # Service URLs (for Docker/production)
 VERIFIER_URL=http://127.0.0.1:3002
+# Verifier warm-up endpoint used by frontend banner
+# Default: see DEPLOY.md (used for frontend warm-up banner)
+NEXT_PUBLIC_VERIFIER_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_VERIFIER_URL key should go before the VERIFIER_URL key

(UnorderedKey)